CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 
 - [web] Keep User-Agent to the URI after an OIDC login to Monocle.
 - [nix] The project is now available as an installable flake. The file structure has been updated. Development environment is provided by `nix develop` instead of `nix-shell`.
+- [compose] Bumped Elasticsearch to 7.17.5.
```

Comment:
Is there a special reason for the bump, or is this just regular maintenance?

Reply:
7.10.1 is EOL 2022-05-11 (https://www.elastic.co/fr/support/eol) and maybe before moving to 8.X that might be better to ensure we run the most recent 7.X version.